src/test/resources/unimplemented_go/floats/float.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+
+}
```

Comment:
This file is not testing anything. Besides, floats have been added in Gobra in the meanwhile.

Reply:
It now tests the parser for (non-hex) floats.